### PR TITLE
docs(e2e-test): explain how to set remote config defaults for e2e tests

### PR DESCRIPTION
**Pull request by cajubelt**
### Description

Somehow it took me a while to find this, so I figured I'd add docs for it.

### Other changes

na

### Tested

na

### How others should test

na

### Related issues

na

### Backwards compatibility

na

### CODE REVIEW

**Review by MuckT — APPROVED** [review]
Thanks for updating!

File: e2e/README.md [hunk 1]
```diff
@@ -196,6 +196,16 @@ class ExampleInput extends React.Component {
 
 It is recommended to follow the scheme parentID/ChildDescription.
 
+## Setting remote config defaults for e2e tests
+
+We do not read remote config values from firebase for e2e tests, so default remote config values are used. A custom set of remote config
+defaults may be needed for some tests which differs from production configuration. For example, in production code,
```
Comment (MuckT) [inline e2e/README.md:202]: 'which differs from production configuration' to 'which differ from the production configuration'

File: e2e/README.md [hunk 2]
```diff
@@ -196,6 +196,16 @@ class ExampleInput extends React.Component {
...
+we do not wish to show FiatConnect cash-outs by default, but in order to test FiatConnect cash-outs end-to-end, we need
```
Comment (MuckT) [inline e2e/README.md:203]: 'but in order to' to 'but to'

**Review by MuckT — APPROVED** [review]
🚀 thanks! Small nits on tone.

File: e2e/README.md (hunk 1 above)
Comment (cajubelt) [reply e2e/README.md:202]: actually mine has the correct grammar

Comment (cajubelt) [reply e2e/README.md:202]: "set" is the subject, which is singular

File: e2e/README.md (hunk 2 above)
Comment (cajubelt) [inline e2e/README.md:203]:
```suggestion
we do not wish to show FiatConnect cash-outs by default, but to test FiatConnect cash-outs end-to-end, we need
```

Comment (cajubelt) [reply e2e/README.md:203]: in the future @MuckT can you leave docs suggestions like this^ so I can accept them with a click?

**Pull request merged**